services/system/AuthEcSys/include/services/system/AuthEcSys.hpp

```diff
@@ -7,38 +7,71 @@
 
 namespace SystemService
 {
-   struct AuthRecord
-   {
-      psibase::AccountNumber account;
-      psibase::PublicKey     pubkey;
+   namespace AuthEc
+   {  // Using a namespace here disambiguates this class from the one in AuthSys so docs don't get confused
+      /// A record containing the authorization claims needed for an account using this auth service.
+      struct AuthRecord
+      {
+         /// The account whose transactions will be required to contain the specified public key.
+         psibase::AccountNumber account;
+         
+         /// The public key included in the claims for each transaction sent by this account.
+         psibase::PublicKey     pubkey;
 
-      auto byPubkey() const { return std::tuple{pubkey, account}; }
-   };
-   PSIO_REFLECT(AuthRecord, account, pubkey)
+         auto byPubkey() const { return std::tuple{pubkey, account}; }
+      };
+      PSIO_REFLECT(AuthRecord, account, pubkey)
+   } // namespace AuthEc 
 
+   /// The `auth-ec-sys` service is a default auth service that can be used to authenticate actions for accounts.
+   ///
+   /// Any account using this auth service must store in this service an ECDCSA public key that they own.
+   /// This service will ensure that the specified public key is included in the transaction claims for any
+   /// transaction sent by this account. 
+   ///
+   /// This service only supports K1 keys (Secp256K1) keys.
    class AuthEcSys : public psibase::Service<AuthEcSys>
    {
-     public:
+   public:
       static constexpr auto service = psibase::AccountNumber("auth-ec-sys");
-      using AuthTable = psibase::Table<AuthRecord, &AuthRecord::account, &AuthRecord::byPubkey>;
+      using AuthTable = psibase::Table<AuthEc::AuthRecord, &AuthEc::AuthRecord::account, &AuthEc::AuthRecord::byPubkey>;
       using Tables    = psibase::ServiceTables<AuthTable>;
 
+      /// This is an implementation of the standard auth service interface defined in [SystemService::AuthInterface].
+      /// It will be automatically called by `transact-sys` when it is processing a transaction from a user who
```

Comment:
"it" used twice in the same sentence with two difference antecedents is confusing.

Reply:
Reworded